### PR TITLE
asserts,boot,secboot: switch to a secboot version measuring classic

### PR DESCRIPTION
**Pull request by pedronis**
To have the information available when we seal capture classic in
modeenv and bootchains as we do for other mode characteristics

As for now we assume we don't want so support classic/core remodels

Some things need to be tested but it's best if the tests are added
when we are actually looking at the full picture of installing classic
systems with modes

### CODE REVIEW

File: asserts/model.go [hunk 1]
```diff
@@ -369,6 +369,8 @@ var gradeToCode = map[ModelGrade]uint32{
 	ModelDangerous:  0x10000,
 	ModelSigned:     0x80000,
 	ModelSecured:    0x100000,
+	// reserved by secboot to measure classic models
+	// "ClassicModelGradeMask": 0x80000000
```
Comment (mvo5) [inline asserts/model.go:373]: \o/ Thank you

**Review by mvo5 — APPROVED** [review]
Thank you

**Review by alfonsosanchezbeato — APPROVED** [review]
LGTM, thanks!

**Comment by alfonsosanchezbeato** [comment]
I have just proposed https://github.com/snapcore/secboot/pull/213 to the `uc22` branch that backports support for arm64/riscV, could we have that one as well?

**Comment by pedronis** [comment]
> I have just proposed https://github.com/snapcore/secboot/pull/213 to the uc22 branch that backports support for arm64/riscV, could we have that one as well?

as we discussed some of that code is not even on master yet, so I think it will need to land separately

**Review by mardy — APPROVED** [review]
LGTM, thanks!

**Pull request merged**